monai/data/csv_saver.py

```diff
@@ -48,6 +49,8 @@ def __init__(
                 otherwise, will append new content to the CSV file.
             flush: whether to write the cache data to CSV file immediately when `save_batch` and clear the cache.
                 default to False.
+            delimiter: the delimiter character in the saved file, default to ",", for more details:
+                https://docs.python.org/3/library/csv.html#csv.Dialect.delimiter.
```

Comment:
any reason not using the build-in python csv lib here? the url is from python csv may be it'll cause confusion because the implementation doesn't use it...

Reply:
Hi @wyli , I don't quite remember the initial reason, seems we suppose to support not only `.csv` format?
Because actually users can set `filename` with other ext here.
Let me try to explain more in the doc-string?

Thanks.

Reply:
Hi @wyli ,

I updated the PR in the latest commit.
Could you please help take a look again?

Thanks in advance.

Reply:
Thanks, it looks good to me.